packages/sanity/src/_internal/cli/commands/app/startCommand.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  type CliCommandArguments,
+  type CliCommandContext,
+  type CliCommandDefinition,
+} from '@sanity/cli'
+
+import {type StartPreviewServerCommandFlags} from '../../actions/preview/previewAction'
+import {isInteractive} from '../../util/isInteractive'
+import {getDevAction} from '../dev/devCommand'
+
+const helpText = `
+Notes
+  Changing the hostname or port number might require a new CORS-entry to be added.
+
+Options
+  --port <port> TCP port to start server on. [default: 3333]
```

Comment:
Might be a good quality of life of improvement if it's different port from studio but if it's too much work than don't worry about it.

```suggestion
  --port <port> TCP port to start server on. [default: 3334]
```

Reply:
Yeh, I agree this would be a nice fast-follow improvement